app/src/main/java/com/example/project_1/ui/viewUser/ViewUserDetailFragment.kt

```diff
@@ -49,6 +51,12 @@ class ViewUserDetailFragment : Fragment() {
             val mainActivity = activity as? MainActivity
             mainActivity?.loadFragment(AddUserDetailFragment())
         }
+
```

Comment:
Create viewmodel object with the help of ViewModelProvider and use viewmodel factory to pass dao object.